src/PotentialCalculation/PotentialCalculationSetup/PotentialCalculationSetupCartesian3D.jl

```diff
@@ -57,15 +57,15 @@ function set_point_types_and_fixed_potentials!(point_types::Array{PointType, 3},
     nothing
 end
 
-function fill_ρ_and_ϵ!(ϵ::Array{T}, ρ_tmp::Array{T}, q_eff_fix_tmp::Array{T}, 
+function fill_ρimp_ϵ_ρfix(ϵ::Array{T}, ρ_imp_tmp::Array{T}, ρ_eff_fix_tmp::Array{T}, 

```

Comment:
The order of the arguments of this function is not consistent with the output of `get_ρimp_ϵ_ρfix`.
Should we change the order of the arguments or adjust the function name, i.e. `fill_ϵ_ρimp_ρfix`?